README.md

```diff
@@ -120,6 +120,70 @@ async function find (entityId) {
 }
 ```
 
+## How to use it - Step 2 (Hapi users)
+
+Use the plugin provided by the library before the first route that needs to have access to request ids. Note that some plugins may cause CLS context to get lost. To avoid such issues, you should use any third party plugins that does not need access to request ids *before* you register this plugin.
+
+```javascript
+const Hapi = require('@hapi/hapi')
+const rTracer = require('cls-rtracer')
+
+const init = async () => {
+  const server = Hapi.server({
+    port: 3000,
+    host: 'localhost'
+  })
+  // any third party plugins that does not need access to request ids goes here
```

Comment:
A typo: should be `plugin` instead of `plugins`

Reply:
Will correct, thanks